### PR TITLE
ref(repo): Add `@sentry-internal` namespace to rrweb package names

### PR DESCRIPTION
**Pull request by Lms24**
This PR renames the rrweb packages by prepending our `@sentry-internal` namespace. I opted to take the internal one over the usual @sentry namespace as I find it more fitting for the purpose we want to release these packages for. 

Note: I'm opening this PR against a `publish-fork` branch so we can continue adding fixes and patching rrweb manually until these packages are released.

ref #15

### CODE REVIEW

File: packages/rrdom/rollup.config.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ for (let config of baseConfigs) {
           name: config.name,
           format: 'iife',
           file: pkg.unpkg.replace(pkg.name, config.path),
+          extend: true,
```
Comment (Lms24) [inline packages/rrdom/rollup.config.js:58]: This is the only deliberate change I made: After changing the namespace, rollup threw an error that the package name is illegal unless I add this option. Tbh I'm not entirely sure what the problem here is but I don't think we need this particular build config anyway.

**Comment by Lms24** [comment]
Hmm looks like prettier wasn't run on a lot of files... should we discard the autoformat changes?

File: packages/rrweb-player/src/Controller.svelte [hunk 2]
```diff
@@ -432,9 +348,103 @@
...
             68 0 18.4 18.4 18.4 48.8 0 68L231.2 860z"
-            p-id="1286" />
+            p-id="1286"
+          />
         </svg>
       </button>
     </div>
   </div>
 {/if}
+
```
Comment (mydea) [inline packages/rrweb-player/src/Controller.svelte:358]: Where does this come from 😅

Comment (mydea) [reply packages/rrweb-player/src/Controller.svelte:358]: ah, nevermind, from the top of the file

**Comment by Lms24** [comment]
>Or, maybe we make one PR first with just the prettier changes, then a second one with the package changes?

Sounds good to me 👍

**Review by mydea — APPROVED** [review]

**Review by billyvg — APPROVED** [review]
using the internal namespace makes sense to me, we are not intending for the general public to be using this package.

**Pull request merged**